.github/workflows/pika.yml

```
@@ -201,19 +204,18 @@ jobs:
           ./pikatests.sh all clean
 
       - name: Start codis, pika master and pika slave
-        working-directory: ${{ github.workspace }}/build
-        run: |
-          chmod +x ../tests/integration/start_master_and_slave.sh
-          ../tests/integration/start_master_and_slave.sh
-          chmod +x ../tests/integration/start_codis.sh
-          ../tests/integration/start_codis.sh
+        working-directory: ${{ github.workspace }}
+        run: | 
```

yamllint

[error] trailing spaces

(trailing-spaces)